File: www/node/test/p138_2.py
import numpy as np # 整数をカウントするための関数呼び出し
import numpy.random as rd # 乱数を発生させる関数の呼び出し
saikoro = rd.randint(1, 6+1, 100) # サイコロを 100 回振る
print(saikoro) # 乱数の値を表示
deme = [ ] # 出目の数を数える配列
for i in range(6):
 deme.append(np.count_nonzero(saikoro==i+1)) # 出現回数を数えて配列に追加
print(" 出現数 :",deme) # 乱数の値を表示

File: www/pyjstest/p138_2.js
define('__main__',function (require,exports,module) {
  var PYLIB=require('http://localhost/runtime/lib/python/PyLib.js');
  var __top=PYLIB.moduleScope(PYLIB, false);
  PYLIB.LoopChecker.reset();
  __top.np=require('http://localhost/runtime/lib/python/py_numpy.js').install(PYLIB);
  __top.rd=require('http://localhost/runtime/lib/python/py_numpy,random.js').install(PYLIB);
  __top.saikoro=__top.rd.__getattribute__('randint')(1,(6).__add__(1),100);
  PYLIB.print(__top.saikoro);
  __top.deme=[];
  for (__top.i of __top.range(6)) {
    PYLIB.LoopChecker.check();
    __top.deme.__getattribute__('append')(__top.np.__getattribute__('count_nonzero')((__top.saikoro).__eq__((__top.i).__add__(1))));
  }PYLIB.print(" 出現数 :",__top.deme);
});
requirejs(['__main__'],function(){
            if (window.parent && window.parent.sendResult) {
                window.parent.sendResult($("#output").text(),"py");
            }});
